### PR TITLE
added method to randomly place all ships

### PR DESCRIPTION
**Pull request by MichaelHilton**

### CODE REVIEW

**Comment by MichaelHilton** [comment]
@nelsonni I think you should look at this

**Comment by nelsonni** [comment]
@MichaelHilton No description. What is it about?!

**Comment by MichaelHilton** [comment]
This code was hard to write, it should be hard to understand

**Comment by MichaelHilton** [comment]
This functionality should be in the model, not in the ship class

File: src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java [hunk 1]
```diff
@@ -154,6 +154,79 @@ else if (computer_battleship.scan(coor)){
...
+        randRow = random.nextInt(10 ) + 1;
+        randCol = random.nextInt(10 ) + 1;
+
+        startCoor = new Coordinate(randRow,randCol);
+        if(Math.random() < 0.5) {
+            endCoor = new Coordinate(randRow,randCol+3);
+        }else{
+            endCoor = new Coordinate(randRow+3,randCol);
+        }
+        this.getShip("cruiser").setLocation(startCoor,endCoor);
```
Comment (nelsonni) [inline src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java:200]: This is repeated for each ship. Seems like a code smell.

**Pull request closed without merging**